modules/web/themes/helium/com.haulmont.sample.petclinic/helium-ext-variant-dark-purple-menu.scss

```diff
@@ -0,0 +1,77 @@
+/* Define your theme modifications inside next mixin */
+@mixin com_haulmont_sample_petclinic-helium-ext {
+    --hell-lila: #867AEB;
+    --white: #FFFFFF;
+
+      /* Basic */
+    --primary-color: #6C5CE7;
+    --primary-color_rgb: 108, 92, 231;
+    --secondary-color: #FDD58D;
```

Comment:
Secondary color is used in many places, e.g. for tooltips and system notifications, so yellow color isn't best option for it.

Reply:
As you mentioned that already before in the Slack channel I asked Alina about which color to pick and she said the yellow would be the fine - this is why I kept it like that. I double checked those cases and in fact it had some problems with the font color. I just changed the font color to be dark and now it actually looks good IMHO.